multiple_pipelines_example/utils/cp_parallel.py

```diff
@@ -58,7 +58,7 @@ def run_cellprofiler_parallel(
     commands = []
 
     # make logs directory
-    log_dir = pathlib.Path("./logs")
+    log_dir = pathlib.Path("/home/maggiekeating/Illumination-correction-benchmarking/multiple_pipelines_example/outputs/logs_IC")
```

Comment:
I am curious about this change in the utility file. Is this change because you would like to have log files included in the outputs folder?

Also, if you change this to a whole path, then this will be the path that the log files go to when you use this to run segmentation and feature extraction.

I have two ideas for this here:

1. You can make this a relative path where if you always plan to put the logs in a `output` folder and always run the notebook in a `notebook` folder, then you can change the path to a relative path like this:
```suggestion
    log_dir = pathlib.Path("../outputs/logs")
```
By making it relative, then it will not be only added to the IC folder. Also, removing the `IC` part of the logs will also prevent it from being included when running the segmentation/feature extraction pipelines later.

2. Another idea is to update the `run_cellprofiler_parallel` function into include a new parameter called `logs_dir` so that a user can set the path to where they want the logs to be saved. Here is an example:
```python
def run_cellprofiler_parallel(
    plate_info_dictionary: dict,
    run_name: str,
    log_dir: pathlib.Path
) -> None:
```
You would still need to include the `os.makedirs(log_dir, exist_ok=True)` to make sure that the log directory does exist.

Let me know if you have any questions about this!

Reply:
Yes, I wanted to put the logs in an output file so everything is really easy to find. I hadn't considered the next step will use cp_parallel. Thank you for taking the time to explain this, I really appreciate it! I am going to use your first suggestion since it seems like a very simple fix to this issue.